Utilities/CMake/CMakeLists.txt

```diff
@@ -130,6 +130,15 @@ set(XMLHttp_HTTP_Source_Files
     ../../../Source/HTTP/XMLHttp/xmlhttp_http_task.h
     )
 
+set(XCurl_Source_Files
```

Comment:
>X [](http://example.com/codeflow?start=4&length=1)

Per @jasonsandlin recommendation, dropping 'X' from paths, filenames, and types as this provider could be used with other Curl implementations as well